### PR TITLE
style footer

### PR DESCRIPTION
**Pull request by rccsousa**
Why:
- Page layout was missing a footer and respective styling.
How:
- Updated footer and added styling

### CODE REVIEW

File: src/app/_components/Footer/styles.module.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.footer {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(4, 0.5fr);
+  font-family: Colfax, sans-serif;
```
Comment (davelange) [inline src/app/_components/Footer/styles.module.css]: You have a variable for this 😉

File: src/app/_components/Footer/styles.module.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  padding: 0 10px 0 10px;
+}
+
+.footer h5 {
+  color: var(--sub-blue-700);
+  margin-bottom: 20px;
+}
+
+.item {
+  padding-top: 40px;
```
Comment (davelange) [inline src/app/_components/Footer/styles.module.css]: This could be defined with the row-gap on .footer, right?

Comment (rccsousa) [reply src/app/_components/Footer/styles.module.css]: right! 
i think i'm still mixing padding with gap between things. :\

File: src/app/_components/Footer/styles.module.css (hunk 1 above)
Comment (rccsousa) [reply src/app/_components/Footer/styles.module.css]: you mean the typography or encapsulating the grid as a whole?

Comment (davelange) [reply src/app/_components/Footer/styles.module.css]: I mean just the font-family

File: src/app/_components/Footer/styles.module.css (hunk 2 above)
Comment (davelange) [reply src/app/_components/Footer/styles.module.css]: No stress :)

File: src/app/_components/Footer/styles.module.css (hunk 1 above)
Comment (rccsousa) [reply src/app/_components/Footer/styles.module.css]: fixed! ✅ 
also fixed sizing, titles and text were too big compared to figma / current live version

File: src/app/_components/Footer/styles.module.css (hunk 2 above)
Comment (rccsousa) [reply src/app/_components/Footer/styles.module.css]: fixed, removed item altogether ✅

**Review by davelange — APPROVED** [review]

**Pull request merged**